### PR TITLE
Feature/docs 1950 all-in-one installer

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/updating-migrating/node-artifact-versions.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 This document lists available [patch versions](versioning-policy.md#version-format) of Wallarm node 4.6 in different form-factors. You can track new patch version releases and plan timely upgrades based on this document.
 
+## All-in-one installer
+
+History of updates simultaneously applies to the x86_64 and ARM64 (beta) versions of [all-in-one installer](../installation/nginx/all-in-one.md).
+
+### 4.6.12 (2023-06-30)
+
+* Initial public release 4.6, [see changelog](what-is-new.md)
```
Comment (AnastasiaTWW) [inline docs/latest/updating-migrating/node-artifact-versions.md:11]: it is initial release of all-in-one installer, not 4.6. we did not publish smth related to all-ine-one installer before and the changes listed below are extra. users do not know anything about 4.6.6

File: docs/latest/admin-en/installation-postanalytics-en.md [hunk 2]
```diff
@@ -76,25 +80,57 @@ To install node, you will need a Wallarm token of the [appropriate type][wallarm
 
 To install postanalytics separately via all-in-one script, use:
 
-```
-# If using the x86_64 version:
-sudo sh ./wallarm-4.6.11.x86_64-glibc.sh postanalytics
+    === "API token"
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/installation-postanalytics-en.md:83]: what are indents here for?

File: docs/latest/admin-en/installation-postanalytics-en.md [hunk 3]
```diff
@@ -76,25 +80,57 @@ To install node, you will need a Wallarm token of the [appropriate type][wallarm
...
+        sudo sh wallarm-4.6.12.x86_64-glibc.sh postanalytics
+
+        # If using the ARM64 version:
+        sudo sh wallarm-4.6.12.aarch64-glibc.sh postanalytics
+        ```
 
 ### 4. Install the NGINX-Wallarm module on a separate server
 
-Once the postanalytics module is installed on the separate server, install the NGINX-Wallarm module on a different server:
+Once the postanalytics module is installed on the separate server
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/installation-postanalytics-en.md:105]: punctuation in the end

File: docs/latest/admin-en/installation-postanalytics-en.md [hunk 4]
```diff
@@ -76,25 +80,57 @@ To install node, you will need a Wallarm token of the [appropriate type][wallarm
...
+        # If using the x86_64 version:
+        sudo sh wallarm-4.6.12.x86_64-glibc.sh filtering
+
+        # If using the ARM64 version:
+        sudo sh wallarm-4.6.12.aarch64-glibc.sh filtering
+        ```
+
+1. Perform the after-installation steps, such as enabling analyzing the traffic, restarting NGINX, configuring sending traffic to the Wallarm instance, test and fine tune, as described here:
+
+    * [NGINX stable](../installation/nginx/dynamic-module.md)
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/installation-postanalytics-en.md:131]: these links are wrong, it is not installation from individual pakcages

File: docs/latest/installation/nginx/all-in-one.md [hunk 5]
```diff
@@ -42,11 +42,11 @@ In comparison to the individual Linux packages offered by Wallarm for [NGINX](dy
...
-<! --![!All-in-one compared to manual](../../images/installation-nginx-overview/manual-vs-all-in-one.png) -->
+![!All-in-one compared to manual](../../images/installation-nginx-overview/manual-vs-all-in-one.png)
 
 ## Requirements
 
-* Access to the account with the **Administrator** role in Wallarm Console for the [US Cloud](https://us1.my.wallarm.com/) or [EU Cloud](https://my.wallarm.com/)
+* Access to the account with the **Administrator** role in Wallarm Console for the [US Cloud](https://us1.my.wallarm.com/) or [EU Cloud](https://my.wallarm.com/).
 * Supported OS:
 
     * Debian 10, 11 and 12.x
```
Comment (AnastasiaTWW) [inline docs/latest/installation/nginx/all-in-one.md:52]: check punctuation in this list

File: docs/latest/updating-migrating/node-artifact-versions.md [hunk 6]
```diff
@@ -2,6 +2,19 @@
 
 This document lists available [patch versions](versioning-policy.md#version-format) of Wallarm node 4.6 in different form-factors. You can track new patch version releases and plan timely upgrades based on this document.
 
+## All-in-one installer
+
+History of updates simultaneously applies to the x86_64 and ARM64 (beta) versions of [all-in-one installer](../installation/nginx/all-in-one.md).
```
[the remark this answers is not in the document]
Comment (MaximBashurov) [reply docs/latest/updating-migrating/node-artifact-versions.md:7]: Took as checklist item in DOCS-2175

**Pull request merged**